test/test_site/site.json

```diff
@@ -34,5 +34,14 @@
   "deploy": {
     "message": "Site Update."
   },
-  "headingIndexingLevel": 4
+  "headingIndexingLevel": 4,
+  "plugins" : [
+    "testPlugin"
+  ],
+  "context" : {
+    "testPlugin" : {
+        "pre": "<p>Prerender</p>",
```

Comment:
MarkBind already converts normal text to `<p>` anyway, so it isn't immediately clear that the plugin was the one responsible for doing that.

Maybe use a different tag? `<h2>`?